### PR TITLE
(DOCS-3830) Getting started for MSPs

### PR DESCRIPTION
**Pull request by hestonhoffman**
See [DOCS-3830](https://datadoghq.atlassian.net/browse/DOCS-3830)

Adds a Partners index and getting started guide

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/partners/laying-the-groundwork.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+Datadog does not recommend separating development, test, and production resources. The recommended approach is to manage all resources in the same Datadog organization and delineate the environments through tags. For more information, see [Tagging strategy](#tagging-strategy).
+
+## Custom subdomains
+
+To improve your Datadog experience when handling a large number of organizations, use the custom subdomain feature.
+
+By default, any Datadog organization is accessed through Datadog's access pages, [https://app.datadoghq.com][3] and [https://app.datadoghq.eu][4]. However, custom subdomains can provide a unique URL for each sub-organization. For example, `https://account-a.datadoghq.com`. 
+
+For more information, see [Custom sub-domains][5]).
```
Comment (KateYoak) [inline content/en/partners/laying-the-groundwork.md:46]:
```suggestion
For more information, see [Custom sub-domains][5].
```

File: content/en/partners/laying-the-groundwork.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
+---
+title: Laying the groundwork
+kind: documentation
+description: "How to get started and which key decisions you should make at the very beginning."
+private: true
+---
+
+Welcome to getting started for managed service providers. The first part of the guide covers key decisions you should make at the very beginning of your Datadog journey.
```
Comment (KateYoak) [inline content/en/partners/laying-the-groundwork.md:8]:
```suggestion
This article covers the key decisions you should make at the very beginning of your journey as a Datadog Managed Service Provider.
```

File: content/en/partners/laying-the-groundwork.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+## Key considerations for managed service providers
+
+The way you as a service provider get started with Datadog depends on your business model and your operational model:
+- **Business model**: A key question to answer is whether you are planning on giving your clients their own access to Datadog or not. If you do choose to give clients access to Datadog, set up a multi-organization account to keep client data separate and private.
+- **Operational model**: Another key consideration is whether your client base consists of many homogeneous clients, where programmatic management of many similar-looking Datadog organizations is more important, or whether your clients are fewer or more heterogeneous.
+
```
Comment (KateYoak) [inline content/en/partners/laying-the-groundwork.md:15]: Having considered the above, you are ready to lay the groundwork for your MSP setup with Datadog.

Comment (KateYoak) [inline content/en/partners/laying-the-groundwork.md:15]:
```suggestion
Having considered the above, you are ready to lay the groundwork for your MSP setup with Datadog.
```

File: content/en/partners/delivering-value.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+
+Datadog offers various channels through which you or your clients can notify users about important alerts:
+
+- Email notifications
+- Integrations, such as:
+  - [Slack][24]
+  - [PagerDuty][25]
+  - [Flowdock][26]
+  - [ServiceNow][27]
+  - [Google Hangouts Chat][28]
```
Comment (cswatt) [inline content/en/partners/delivering-value.md:79]:
```suggestion
  - [Google Chat][28]
```

rip hangouts

File: content/en/partners/delivering-value.md [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+  - [Microsoft Teams][29]
+  - And [many more][19]
+
+You can also invoke any REST API using the generic [Webhooks integration][30]. You use a Webhooks integration to not only notify users but also to trigger automatic remediation workflows.
```
Comment (cswatt) [inline content/en/partners/delivering-value.md:83]:
```suggestion
You can also invoke any REST API using the generic [Webhooks integration][30]. You use a Webhooks integration to not only notify users, but also to trigger automatic remediation workflows.
```

File: content/en/partners/delivering-value.md [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+
+To find out more about notifications, see:
+- [Notifications][31]
+- [Send SMS Alerts with Webhooks and Twilio][32] (Blog)
+
+## Setting up visualizations with Dashboards
```
Comment (cswatt) [inline content/en/partners/delivering-value.md:89]:
```suggestion
## Setting up visualizations with dashboards
```

File: content/en/partners/delivering-value.md [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+- Use the [Datadog Notebooks][37] feature to gather data in an exploratory fashion and draft dashboards
+- [Monitor Datacenters and Networks with Datadog][38] (Blog)
+- [Use Associated Template Variables][39] (Blog)
+- [The Datadog Dashboard API][40]
+- [Configure Observability as Code with Terraform & Datadog][41] (HashiCorp webinar)
+
+### Visualizations for users without Datadog access
+
+Depending on your business model, your clients might not require access to Datadog themselves. However, you might still want to provide Datadog visualizations to your clients. You have the following options for providing Datadog visualizations:
+- [Dashboard sharing][42]: Provide a status page to your clients by sharing a public URL to a read-only dashboard, or share the dashboard privately using an individual e-mail address.
```
Comment (cswatt) [inline content/en/partners/delivering-value.md:117]:
```suggestion
- [Dashboard sharing][42]: Provide a status page to your clients by sharing a public URL to a read-only dashboard, or share the dashboard privately using an individual email address.
```

File: content/en/partners/laying-the-groundwork.md [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+## Key considerations for managed service providers
+
+The way you as a service provider get started with Datadog depends on your business model and your operational model:
+- **Business model**: A key question to answer is whether you are planning on giving your clients their own access to Datadog or not. If you do choose to give clients access to Datadog, set up a multi-organization account to keep client data separate and private.
+- **Operational model**: Another key consideration is whether your client base consists of many homogeneous clients, where programmatic management of many similar-looking Datadog organizations is more important, or whether your clients are fewer or more heterogeneous.
+Having considered the above, you are ready to lay the groundwork for your MSP setup with Datadog.
+## Prerequisites
+
+Before working on implementing Datadog as a Service Provider, it is recommended that you complete the
```
Comment (cswatt) [inline content/en/partners/laying-the-groundwork.md:18]:
```suggestion
Before working on implementing Datadog as a service provider, it is recommended that you complete the
```

File: content/en/partners/laying-the-groundwork.md [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+<a href="https://partners.datadoghq.com/prm/English/c/technical_content" target="_blank">Datadog Technical Specialist training</a>.
+
+The training and certification familiarize you with many of the topics covered in the next chapters, enabling you to get started immediately.
+
+## Organization setup
+
+One of the key decisions for a Service Provider to make is how to set up client Datadog accounts, called "Organizations" (or "orgs" for short). While users can be associated with more than one organization, monitored resources are tied to a single organization. Choosing the right organization structure from the beginning helps to quickly create value for you and your clients.
```
Comment (cswatt) [inline content/en/partners/laying-the-groundwork.md:25]:
```suggestion
One of the key decisions for a service provider to make is how to set up client Datadog accounts, called "organizations" (or "orgs" for short). While users can be associated with more than one organization, monitored resources are tied to a single organization. Choosing the right organization structure from the beginning helps to quickly create value for you and your clients.
```

File: content/en/partners/laying-the-groundwork.md [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+
+### Single-organization or multiple-organization
+
+Datadog offers the possibility of managing multiple child organizations from one parent organization. This is the typical deployment model used by MSPs to prevent clients from having access to each others' data. In a multi-org setup, one child organization is created for each client, and the client is restricted to their own child organization. 
+
+Use a single-org setup if you have no plans to give your clients access to Datadog and do not have a strict requirement to separate client data.
+
+For more information about Organization Management, see the [Managing Multiple Organization Accounts][1] documentation.
```
Comment (cswatt) [inline content/en/partners/laying-the-groundwork.md:33]:
```suggestion
For more information about organization management, see the [Managing Multiple Organization Accounts][1] documentation.
```

File: content/en/partners/laying-the-groundwork.md [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+
+To improve your Datadog experience when handling a large number of organizations, use the custom subdomain feature.
+
+By default, any Datadog organization is accessed through Datadog's access pages, [https://app.datadoghq.com][2] and [https://app.datadoghq.eu][3]. However, custom subdomains can provide a unique URL for each sub-organization. For example, `https://account-a.datadoghq.com`. 
+
+For more information, see [Custom sub-domains][4]).
+
+## User roles and custom role-based access control (RBAC)
+
+Experience shows that both MSP-internal and client users often do not fall clearly into one of the three [Datadog default roles][5]. It's good practice to create custom roles to limit user permissions in certain areas.
```
Comment (cswatt) [inline content/en/partners/laying-the-groundwork.md:51]:
```suggestion
Experience shows that both MSP-internal and client users often do not fall clearly into one of the three [Datadog default roles][5]. It's a good practice to create custom roles to limit user permissions in certain areas. 
```

File: content/en/partners/laying-the-groundwork.md [hunk 12]
```diff
@@ -0,0 +1,114 @@
...
+
+Experience shows that both MSP-internal and client users often do not fall clearly into one of the three [Datadog default roles][5]. It's good practice to create custom roles to limit user permissions in certain areas. 
+
+For more information, see:
+- [Custom roles][6]
+- [Role Based Access Control][7]
+
+## Single sign-on (SSO) considerations
+
+In a service provider context, you have two considerations for Single sign-on (SSO):
```
Comment (cswatt) [inline content/en/partners/laying-the-groundwork.md:59]:
```suggestion
In a service provider context, you have two considerations for single sign-on (SSO):
```

File: content/en/partners/laying-the-groundwork.md [hunk 13]
```diff
@@ -0,0 +1,114 @@
...
+- Single sign-on for your organization
+- Single sign-on for your clients
+
+Besides the obvious advantage of a unified authentication mechanism, using SAML Single Sign-On also vastly simplifies the user provisioning process. Using SAML allows you to use Just-in-Time (JiT) user provisioning, eliminating the need to manually or programmatically create users.
```
Comment (cswatt) [inline content/en/partners/laying-the-groundwork.md:63]:
```suggestion
Besides the obvious advantage of a unified authentication mechanism, using SAML Single Sign-On also vastly simplifies the user provisioning process. Using SAML allows you to use just-in-time (JiT) user provisioning, eliminating the need to manually or programmatically create users.
```

File: content/en/partners/laying-the-groundwork.md [hunk 14]
```diff
@@ -0,0 +1,114 @@
...
+
+SAML authentication is enabled on a Datadog organization or sub-org, meaning you can have different SAML providers for different sub-orgs. However, this also means that if you have two groups of users with different SAML providers, those users have to be in separate orgs. Make sure you think about SAML authentication when when planning your multi-organization setup.
```
Comment (cswatt) [inline content/en/partners/laying-the-groundwork.md:65]:
```suggestion
SAML authentication is enabled on a Datadog organization or sub-org, meaning you can have different SAML providers for different sub-orgs. However, this also means that if you have two groups of users with different SAML providers, those users have to be in separate orgs. Make sure you think about SAML authentication when planning your multi-organization setup.
```

File: content/en/partners/laying-the-groundwork.md [hunk 15]
```diff
@@ -0,0 +1,114 @@
...
+
+### User training
+
+Datadog's goal is to provide a service that is easy and intuitive to use. Experience shows that most users feel comfortable working with the product, and learning as they go along.
+
+Here are some useful resources for users that prefer to have training on the most important aspects of the product:
+
+- [Datadog's YouTube channel][13]: With introduction videos posted whenever new features are released as well as videos on Tips & Tricks and Best Practices, Datadog's YouTube channel is a great source for high-level training.
+- [Datadog Learning Center][14]: The Datadog Learning Center is a great way for users to get to know the platform in-depth. When signing up for the Learning Center, a Datadog sandbox environment is automatically provisioned free of charge, allowing users to play around with the product without fear of breaking anything.
+- [The Datadog Blog][15]: With over 700 entries, The blog is a key source of information on how to use Datadog to monitor key services, tools, and technologies in your client environments, as well as information on the latest product releases.
```
Comment (cswatt) [inline content/en/partners/laying-the-groundwork.md:89]:
```suggestion
- [The Datadog Blog][15]: With over 700 entries, the blog is a key source of information on how to use Datadog to monitor key services, tools, and technologies in your client environments, as well as information on the latest product releases.
```

**Review by cswatt — APPROVED** [review]

**Pull request merged**